### PR TITLE
Add a test case for bats

### PR DESCRIPTION
**Pull request by Willamin**

### CODE REVIEW

File: source/scripts/tests/BatTest.js [hunk 1]
```diff
@@ -30,5 +31,13 @@ export default function BatTest() {
         // being spawned in.
     })
 
-    // Put assertions here that test your code!
+    // Had to use JSON.stringify to compare objects
```
Comment (ehgoodenough) [inline source/scripts/tests/BatTest.js]: Hmm.. We should find a better way of comparing two positions, so we don't have to stringify them again next time.

**Review by ehgoodenough — APPROVED** [review]

File: source/scripts/tests/BatTest.js (hunk 1 above)
Comment (Willamin) [reply source/scripts/tests/BatTest.js]: I wasn't a huge fan either. An alternative I thought about was making a helper function that takes two positions and checks whether `position1.x == position2.x && position1.y == position2.y`. I wasn't sure where would be best to put a helper function like that

**Review by jcpirtle — COMMENTED** [review]
Will, change your stringify to what I have in my test and you should be good to go

**Review by ehgoodenough — CHANGES_REQUESTED** [review]
When we slack-chat'd last Tuesday, we decided to move towards `.eql` instead of `Json.stringify`.

**Comment by ehgoodenough** [comment]
Also, could you please provide a test link? We've been slacking off on providing those.

**Comment by Willamin** [comment]
Build link (`production`): https://Willamin.github.io/enchiridion/bat-test
Build link (`development`): https://willamin.github.io/enchiridion/bat-test-dev/

**Review by ehgoodenough — APPROVED** [review]
LGTM! :D

**Pull request merged**